.github/workflows/deploy.yml

```diff
@@ -60,13 +90,226 @@ jobs:
     - name: Package pre-release binaries
       if: startsWith(github.ref, 'refs/tags/p')
       run: |
-        7z a rav1e-windows.zip "C:\usr\rav1e-windows-sdk"
+        7z a rav1e-windows-${{ matrix.conf }}.zip `
+             "C:\usr\rav1e-windows-${{ matrix.conf }}-sdk"
 
     - name: Package release binaries
       if: startsWith(github.ref, 'refs/tags/v')
       run: |
-        7z a rav1e-${{ steps.tagName.outputs.version }}-windows.zip `
-             "C:\usr\rav1e-windows-sdk"
+        $ZIP_PREFIX = "rav1e-${{ steps.tagName.outputs.version }}-windows"
+        7z a "$ZIP_PREFIX-${{ matrix.conf }}.zip" `
+             "C:\usr\rav1e-windows-${{ matrix.conf }}-sdk"
+
+    - name: Upload rav1e msvc binary
+      if: matrix.conf == 'msvc'
+      uses: actions/upload-artifact@v2
+      with:
+        path: target/release/rav1e.exe
+
+    - name: Upload pre-release binaries
+      if: startsWith(github.ref, 'refs/tags/p')
+      uses: actions/upload-artifact@v2
+      with:
+        path: rav1e-windows-${{ matrix.conf }}.zip
+
+    - name: Upload release binaries
+      if: startsWith(github.ref, 'refs/tags/v')
+      uses: actions/upload-artifact@v2
+      with:
+        path: rav1e-${{ steps.tagName.outputs.version }}-windows-${{ matrix.conf }}.zip
+
+
+  linux-binaries:
+
+    strategy:
+      matrix:
+        target:
+         - x86_64-unknown-linux-musl
+         - aarch64-unknown-linux-gnu
```

Comment:
Perhaps `aarch64-unknown-linux-musl` is preferable as then the binary will run on both Android and arbitrary Linux distributions. However, my experience is that getting a working aarch64 musl cross-compilation environment may require more effort.

Reply:
I've seen an `aarch64-unknown-linux-musl` binary in the last pre-release. Perhaps who had built that could explain us how to do that.